scipy/stats/_binned_statistic.py

```diff
@@ -55,6 +55,9 @@ def binned_statistic(x, values, statistic='mean',
             values, and outputs a single numerical statistic. This function
             will be called on the values in each bin.  Empty bins will be
             represented by function([]), or NaN if this returns an error.
+            If the function is `np.mean`, `np.median`, `np.sum`, `np.std`,
+            `np.min`, or `np.max`, the equivalent string-name statistic
```

Comment:
I'm guessing that most users won't understand the significance of this statement. Maybe add some interpretation - what is the important difference between a "string-name statistic" and the corresponding NumPy function?
 Maybe this means that "code that that is optimized for use with `binned_statistic` will perform the desired operation without calling the NumPy function explicitly"?

Reply:
I've added that text in the latest commit.